grails-app/assets/javascripts/streama/controllers/modal-opensubtitles-ctrl.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+
+angular.module('streama').controller('modalOpensubtitleCtrl', [
+  '$scope', '$uibModalInstance', 'apiService', 'video', 'localStorageService', 'languageData', '$rootScope',
+  function ($scope, $uibModalInstance, apiService, video, localStorageService, languageData, $rootScope) {
+
+    $scope.videoName = video.name || video.title
+    $scope.loading = false;
+    $scope.localFilesEnabled = false;
+    $scope.localFiles = [];
+    $scope.opensubtitles;
+    $scope.isSearch = false;
+    $scope.colMd = 12;
+    $scope.numberOfResults = 0;
+    $scope.opensubtitleLanguages = {
+      availableOptions: languageData,
+      model: languageData[121].IdSubLanguage
+    };
+
+    $scope.activeTab = localStorageService.get('activeFileModalTab') || 'upload';
+    $scope.searchByHash = false
+
+
+    var localFileLastPath = localStorageService.get('localFileLastPath') || '';
+    $scope.localDir = localFileLastPath.split('/') || [];
+    $scope.video = video;
+    $scope.uploadStatus = {};
+
+    $scope.getSubtitles = getSubtitles;
+    $scope.uploadSubtitles = uploadSubtitles;
+    $scope.cancel = cancel;
+
+    function getSubtitles() {
+      apiService.subtitle.getOpensubtitles(video, $scope.videoName, $scope.opensubtitleLanguages.model, $scope.searchByHash).then(function (data) {
+        $scope.opensubtitles = data.data;
+        $scope.isSearch = true;
+        $scope.colMd = 6;
```

Comment:
I'd prefer a more descriptive variableName than colMd - is this a length? a width? A Counter?